assets/javascripts/discourse/components/topic-chat-float.js

```diff
@@ -124,6 +124,12 @@ export default Component.extend({
     this.switchChannel(channel);
   },
 
+  @discourseComputed("expanded")
+  topLineClass(expanded) {
+    const baseClass = "topic-chat-drawer-header__top-line";
+    return expanded ? baseClass + "--expanded" : baseClass + "--collapsed";
```

Comment:
```suggestion
    return expanded ? `${baseClass}--expanded` : `${baseClass}--collapsed`;
```

Minor but interpolating the strings is easier to read for me.